src/app/index.ts

```diff
@@ -18,9 +23,18 @@ export const app = async (): Promise<void> => {
     res.status(500).json({ message: err.message });
   });
 
-  app.listen(port, () => {
-    console.log(`App is running on port ${port}`);
-  });
+  const options = { useNewUrlParser: true, useUnifiedTopology: true, useFindAndModify: true, useCreateIndex: true };
+
+  mongoose.set('useFindAndModify', false);
+
+  mongoose.connect(url, options).then(() =>
+    app.listen(port, () =>
+      console.log(`Server running on http://localhost:${port} and database is ${url}`)
```

Comment:
@hsedaqat1378 please remove `localhost` and make it dynamic some how you should read it from env file.

Reply:
remove localhost from console.log() ? 
 console.log(`Server running on **http://localhost:**${port} and database is ${url}`)